docs/src/gettingstarted/introduction/template/cli-install.md

```diff
@@ -12,65 +12,18 @@ With all of the requirements met, install the CLI to start developing with Platf
 
 {{< asciinema src="videos/asciinema/verify-cli-extended.cast" >}}
 
-1. **Install the CLI**
+## 1. Install the CLI
 
-    In your terminal, run the following command depending on your operating system:
+{{% cli-installation %}}
 
-{{< codetabs >}}
+## 2. Test the CLI
 
----
-title=Linux/macOS
-file=none
-highlight=bash
-markdownify=false
----
-
-curl -fsS https://platform.sh/cli/installer | php
-
-<--->
-
----
-title=Windows
-file=none
-highlight=bash
-markdownify=false
----
-
-curl -f https://platform.sh/cli/installer -o cli-installer.php
-php cli-installer.php
-
-{{< /codetabs >}}
-
-
-2. **Authenticate and Verify**
-
-   Once the installation has completed, you can run the CLI in your terminal with the command
-
-   ```bash
-   platform
-   ```
-
-   {{< note >}}
-
-   If you opened your free trial account using another login (such as GitHub),
-   you can't authenticate with this command until you setup your account password with Platform.sh in the console.
-
-   {{< /note >}}
-
-   You should now see a list of your Platform.sh projects, including the template you made in this guide.
-   You can copy its *project ID* hash, and then download a local copy of the repository with this command:
-
-   ```bash
-   platform get <project ID>
-   ```
-
-   With a local copy, you can create branches, commit to them, and push your changes to Platform.sh right away!
-
-   ```bash
-   git push platform main
-   ```
+To interact with your projects you need to be logged in.
+To open the authentication page and get a list of your Platform.sh projects, run:
```

Comment:
:hammer_and_wrench: I'd match this to the text above: https://github.com/platformsh/platformsh-docs/pull/2400/files#diff-1adfce48ccf0b087954a418b57ca8268405f68e3c0cdc694ac6cd5d81a0419a7R19-R20 

Seems like the whole thing from `## 1.` through the `platform` command could be in the same short code, but at least the wording here could be aligned.

Reply:
Short code is improved and now has the `platform` command in it as well.